### DISCUSSION TITLE
Add an option to GitHub Actions config (yml) to skip sending email notifications

### DISCUSSION THREAD

**Discussion by barseghyanartur** [community/community#12292] [Actions]
It would be good to add an option to GitHub Actions config (yml) to skip sending email notifications, leaving the email clean and unpolluted.

In that case, whoever would want to see notifications, would have to go to the web interface.

**Comment by Yash-Singh1** [comment]
+1 I need this a lot when I automerge PRs in my repositories.

**Comment by ethomson** [comment]
Thanks for the feedback @barseghyanartur - I think that there may be some areas that we can improve our notifications experience. I'll make sure that we take your feedback into account when we look into improving this area.

Thanks much for taking the time to leave feedback.

**Comment by jeherve** [comment]
Here is a somewhat related suggestion:
https://github.com/github-community/community/discussions/13015
